### DISCUSSION TITLE
JSLT - Is there any block comment syntax?

### DISCUSSION THREAD

**Discussion by cvsudheer108** [schibsted/jslt#225] [Q&A] [answered]
I see that single line comment is working in JSLT with : //
But I found the block comment is not working with : /*. */ OR ```
Please let me know...
Thanks,
Sudheer

**Comment by larsga** [comment, marked as answer]
There is no block comment syntax in JSLT. To be honest I didn't see the point of having one.

**Reply by cvsudheer108** [reply]
Thanks, @larsga. The only scenario is when you have a block of code that you want to comment on and keep for any reason, this will be useful.

**Reply by larsga** [reply]
For me commenting out a block is trivial even with just line comments. Isn't this something IDEs support?

**Reply by cvsudheer108** [reply]
Yes.. I am using VS Code plug-in and that does a good job. Thank you
